src/extensions/technotype/technotypeext.h

```diff
@@ -53,6 +53,12 @@ class TechnoTypeClassExtension final : public Extension<TechnoTypeClass>
 
     public:
 
+        /*
```

Comment:
Comment formatting here is different to the style of the project. Slight copy and paste issue?